StashSearch/StashComponent.cs

```diff
@@ -44,25 +44,25 @@ internal class StashComponent : MonoBehaviour
         private ComplexStashPanel _complexStash;
         private GridView _gridView => _complexStash.GetComponentInChildren<GridView>();
 
+        private bool _hasMovedComplexStash = false;
+        private Vector2 _oldComplexStashSizeDelta;
+        private Vector3 _oldComplexStashLocalPosition;
+
         // Get the session
         public static ISession _session => ClientAppUtils.GetMainApp().GetClientBackEndSession();
 
         public StashComponent()
         {
         }
 
-        private void Start()
+        private void Awake()
```

Comment:
`OnEnable()` might be called before `Start()` so changing to `Awake()` resolves that